content/en/docs/pipelines/tutorials/benchmark-examples.md

```diff
@@ -64,7 +64,7 @@ as an example.
 Use the following instructions to run the benchmark script on your Kubeflow
 Pipelines cluster.
 
-1. Download the the [run_service_api.ipynb](https://storage.googleapis.com/ml-pipeline/sample-benchmark/run_service_api.ipynb)
+1. Download the [run_service_api.ipynb](https://storage.googleapis.com/ml-pipeline/sample-benchmark/run_service_api.ipynb)
    benchmark script in your Jupyter notebook environment.
 1. Open [run_service_api.ipynb](https://storage.googleapis.com/ml-pipeline/sample-benchmark/run_service_api.ipynb)
```

Comment:
We can use the "code font" for file names (e.g. `.ipynb` notebooks and YAML files (search for `.yaml`)):

```suggestion
1. Open [`run_service_api.ipynb`](https://storage.googleapis.com/ml-pipeline/sample-benchmark/run_service_api.ipynb)
```

(Reference - [Google docs style guide: Code in text](https://developers.google.com/style/code-in-text#some-specific-items-to-put-in-code-font)).

Reply:
- Line 87:

```diff
-      [taxi_updated_pool.yaml](https://storage.googleapis.com/ml-pipeline/sample-benchmark/taxi_updated_pool.yaml)
+      [`taxi_updated_pool.yaml`](https://storage.googleapis.com/ml-pipeline/sample-benchmark/taxi_updated_pool.yaml)
```

- Lines 111-112:

```diff
- [`run_service_api.ipynb`](https://storage.googleapis.com/ml-pipeline/sample-benchmark/run_service_api.ipynb)
+ [`run_service_api.ipynb`](https://storage.googleapis.com/ml-pipeline/sample-benchmark/run_service_api.ipynb)
- using the [taxi_updated_pool.yaml](https://storage.googleapis.com/ml-pipeline/sample-benchmark/taxi_updated_pool.yaml)
+ using the [`taxi_updated_pool.yaml`](https://storage.googleapis.com/ml-pipeline/sample-benchmark/taxi_updated_pool.yaml)